providers/os/connection/docker/docker_file_connection.go

```diff
@@ -21,10 +21,14 @@ var _ shared.Connection = &DockerfileConnection{}
 
 type DockerfileConnection struct {
 	*local.LocalConnection
+	// Filename must be the absolute path of the Dockerfile so
+	// that we find the file downstream
 	Filename string
```

Comment:
```suggestion
	FileAbsSrc string
```
i think that makes more sense given the comment above it too